content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -190,6 +192,15 @@ sudo apt-mark hold kubelet kubeadm kubectl
 ```
 {{% /tab %}}
 {{% tab name="CentOS, RHEL or Fedora" %}}
+Please note that Kubernetes provides all official `rpm` packages via
+[yum.k8s.io](https://yum.k8s.io).
+{{< warning >}}
+The `kubernetes-cni` package was obsoleted with the release of Kubernetes
+v1.18.4, v1.17.7 and v1.16.11 on June 17th, 2020. This means in case of
+doing an upgrade from prior versions, please ensure that the package gets
+**manually** removed before doing the upgrade by executing `yum remove
+kubernetes-cni-0.7.5-0`.
+{{</ warning >}}
```

Comment:
/cc @sftim 
PTAL on the wording, AFAIK we do not include dates and PATCH version information in notes/warnings.

Reply:
nit: Again, will this or does this information appear in the release notes?

Reply:
I think we can just state that it is obsolete from now own. We do not mention the obsoletion in the release notes, but I could follow-up with a dedicated PR to add it to them.